### ISSUE TITLE
removeAllNetworks() in .connect()

### ISSUE THREAD

**Issue by slegouffe** [meg768/rpi-wifi-connection#2]
Hi !

I'm really surprise that the connect method call "removeAllNetwork". After testing, it really does : all my configured Wifi Network are deleted in my wpa_supplicant.conf. 
Any chance to remove "removeAllNetworks()" and just call "addNetwork()" ? 

Thanks !

**Comment by meg768** [comment]
I know, it is a destructive thing to do but my thought was that you only need one. May have to re-think here. Default is to just add, option to delete (or disable) all others? But I want to keep everything enclosed in the connect() method since it saves the configuration if everything is successful. If not, it reverts to the original settings. What about connect({ssid:'network', psk:'password', purge:true})? Where purge is default set to false.

**Comment by meg768** [comment]
... or some parameter name like it...

**Comment by meg768** [comment]
Available on npm now.

**Comment by meg768** [comment]
But now, when connecting to the same network with the same ssid/psk it causes duplicates in wpa_supplicant.conf but with the disabled attribute set. What do you think? Is this OK behavior or should the duplicates be removed?

**Comment by meg768** [comment]
Would you ever want two connections to the same ssid?

**Comment by slegouffe** [comment]
Hi ! 
Really nice work ! It works perfectly without duplication of network. Thks a lot !
But there is another issue now...For better clarity I'll explain it in another issue topic ;-)

**Comment by slegouffe** [comment]
Hmmm...after testing, I saw that wpa_cli set each time all previously configured network to "disabled=1". Do you know if there is a wpa_cli option to not add this property ?
Thks !

**Comment by meg768** [comment]
True, calling **wpa_cli select_network** selects the specified network and disables others according to the documentation. Maybe I should try **wpa_cli enable_network**...

**Comment by meg768** [comment]
And I agree, it would really be nice to just move the Raspberry around and it would connect to what ever network it previously has been connected to. Just like my laptop.
